misc/dinit/nix-daemon.service.in

```diff
@@ -0,0 +1,4 @@
+type      		= process
```

Comment:
Is there a reason for the weird spacing? Does dinit require this? (6 spaces followed by 2 tabs here, and 6 spaces followed by 1 tab for the `logfile`)

Reply:
I've just noticed that. To be honest - I've just followed how some built-in service files on Artix look like. I've wanted to keep it the same way, so the `=` is in the same place. (ntpd and cronie service files, for example). I can fix it with next commit. 
![image](https://user-images.githubusercontent.com/30440603/169896829-0e1c16d3-56f5-4501-97e5-28944f42778f.png)
![image](https://user-images.githubusercontent.com/30440603/169896906-6883e97a-32e5-4cbf-956d-0fab5f969ee2.png)

Thanks for noticing. Besides from that, is there anything else? I'll for sure add more service files soon, however, my question is, should I declare some other things somewhere else to make it work with Nix? If so, where? Is the directory where the files are even OK?